### PR TITLE
Adding action hooks before and after the Price Filter Widget input HTML

### LINKED ISSUE #22516: Make the Price Filter Widget great again, with hooks
**Is your feature request related to a problem? Please describe.**
I'm trying to create a Shop Page sidebar with many different filters. One of them is the Price Filter Widget from WooCommerce, and some others are handcrafted.
The Price Filter Widget:
- outputs its own `<form />` tag
- does not offer any hook in this `<form>` element

So we have no way to add other additional filters in the Price Filter Widget form, to use the same submit button.

**Describe the solution you'd like**
It would be great to refactor the Price Filter Widget output so that we can inject custom code inside its `<form>` tag.

**Describe alternatives you've considered**
No "clean" alternative to do that ¯\_(ツ)_/¯

### PR DESCRIPTION
**Pull request by psaikali**
Fixes #22516 

### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:
- Adding a new `woocommerce_price_filter_widget_before_input` action hook
- Adding a new `woocommerce_price_filter_widget_after_input` action hook

Both are inside the native `<form />` element of the Price Filter Widget. These hooks will allow adding new homemade filters inside the same `<form>` of the Price Filter Widget, so that we don't have to create and submit multiple forms to have cumulative filters.

Closes #22516

### CODE REVIEW

**Comment by psaikali** [comment]
Actually, I'm thinking about an alternative: what about using a template file for this widget output?
It would give us way more flexibility.

**Comment by mikejolley** [comment]
See my comment in https://github.com/woocommerce/woocommerce/issues/22516#issuecomment-456410427

tl;dr actions here lack flexibility and there are other ways to include price filter in custom filtering/sidebar.
